coremltools/_deps/__init__.py

```diff
@@ -10,9 +10,8 @@
 import platform as _platform
 import re as _re
 import sys as _sys
-from distutils.version import StrictVersion as _StrictVersion
 
-from packaging import version
+from packaging import version, Version
```

Comment:
This line errors out:
```
ImportError: cannot import name 'Version' from 'packaging'
```

In order to get `Version` (with uppercase V) I think it should be:
```
from packaging.version import Version
```

Not sure if we still need `version` (with a lowercase).

Reply:
```def parse(version: str) -> Version:
    return Version(version)```

version.parse essentially just called the Version ctr, I removed this too for better clarity